sphinx_proof/domain.py

```diff
@@ -37,7 +38,10 @@ def generate(self, docnames=None) -> Tuple[Dict[str, Any], bool]:
         if not hasattr(self.domain.env, "proof_list"):
             return content, True
 
+        # import pdb;
```

Comment:
@AakashGfude is there a reason we are leaving these comments in here? Probably should remove these

Reply:
aaah. my bad.